docs/reference/extra_fields.rst

```diff
@@ -0,0 +1,56 @@
+Extra fields from Solr
+======================
+
+This feature allows you to extract additionally indexed Solr fields from each SearchHit in SearchResult. For example, you can index some fields from children content on the parent content and then get those fields during search (eg. children count).
+
+.. note::
+
+    This feature is available only with the Solr search engine.
+
+1. Usage
+~~~~~~~~
+
+In order for this functionality to work, you have to use overridden `Netgen\EzPlatformSearchExtra\API\Values\Content\Search\Query` or `Netgen\EzPlatformSearchExtra\API\Values\Content\Search\LocationQuery` queries and use it's property `extraFields` provide a list of additional fields that you want to extract from the Solr document. Those fields, if exist, and their values will appear in the `extraFields` property of each `SearchHit` object contained in the `SearchResult.`
```

Comment:
> use it's property `extraFields` **to** provide

Missing "to".

Reply:
@pspanja Fixed in https://github.com/netgen/ezplatform-search-extra/pull/52/commits/dae56dd3ac75ab0e9acdc4bda9dbe7f16a2418ff.